content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "The Same Subject Continued: Concerning Dangers from Foreign Force and Influence"
+date: "2020-11-18"
+originalAuthor: John Jay
+description: "It is better to have a single, united government when it comes to the dangers of war an international relations."
+---
+
+It hardly takes a genius to recognize that the citizens of any country (especially a people as intelligent and well-informed as you, my fellow Americans) rarely adopt incorrect opinions about what is good for them--at least not for very long. So, the fact that we have long been united in believing that we should stay united under a single, sufficiently powerful federal government means that it's probably a good idea.
+
+In fact, the more I look into the reasons for this opinion, the more convinced I am that they are compelling and decisive.
+
+There are lots of different things that a democratic government needs to do, but providing for the safety of her citizens is the first. "Safety" means lots of things to different people and in different situations, so it's important to define what I mean precisely.
+
+For this essay, I am considering safety as it relates to peace from foreign and domestic violence. Let's start with foreign violence. Is it true that a national government really affords the best security against hostilities from abroad?
+
+Let's start with the assumption that wars arise from either real or perceived causes. If this is right, then we should think about whether a united America or a divided America is likely to produce more causes for war with other countries. If it turns out that a united America will give fewer causes for war, then a union will be more likely to lead to peace with other nations.
+
+So, what are the just causes for war? They typically come either from broken treaties or direct violence. America has already formed treaties with six countries--and all but Prussia have strong navies (and so they could attack us). We also have lots of trade with Portugal, Spain, and Britain, which all have colonies on our continent. <!-- This is a pretty big paraphrase -->
```

Comment:
Let's remove HTML comments. I don't know, but they might be rendered as plain text. I would be surprised, but it might happen.

Reply:
I can't imagine that they would, right? This can be the secret content for the people who view source.